hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/balancer/Balancer.java

```diff
@@ -705,7 +705,6 @@ static private int doBalance(Collection<URI> namenodes,
     LOG.info("excluded nodes = " + p.getExcludedNodes());
     LOG.info("source nodes = " + p.getSourceNodes());
     checkKeytabAndInit(conf);
-    System.out.println("Time Stamp               Iteration#  Bytes Already Moved  Bytes Left To Move  Bytes Being Moved");
```

Comment:
Hmm why you think this is not valid? I thought this is the header line for the balancer progress.

Reply:
ok